.github/workflows/ci.yaml

```
@@ -57,7 +57,7 @@ jobs:
         run: pnpm test
 
       - name: Install Playwright Browsers
-        run: pnpx playwright install --with-deps chromium
+        run: pnpm playwright:install
```

I think we don't really have to add a dedicated install script
```suggestion
        run: pnpx playwright install --with-deps chromium
```

---

If we run playwright via pnpx it uses the latest as pnpm contrary to npm seems to not respect versions specified in package.json.

---

nvm pnpm exec respects versions.